lib/dal/tests/integration_test/func/authoring/save_func/attribute.rs

```diff
@@ -448,6 +458,209 @@ async fn delete_add_modify(ctx: &mut DalContext) {
     }
 }
 
+#[test]
+async fn create_attribute_prototype_with_argument(ctx: &mut DalContext) {
```

Comment:
This is the big one. We have to do some gymnastics for the assertions, but it reflects the creation of new attribute bindings when one already exists.